NineChronicles.HttpGateway/Controllers/BalanceController.cs

```diff
@@ -0,0 +1,60 @@
+﻿namespace NineChronicles.HttpGateway.Controllers
+{
+    using System;
+    using System.Text.Json;
+    using System.Threading.Tasks;
+    using Bencodex;
+    using Bencodex.Types;
+    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.Extensions.Logging;
+    using Nekoyume;
+    using Nekoyume.Model.State;
+    using Nekoyume.Shared.Services;
+
+    [ApiController]
+    [Route("[controller]")]
+    public class BalanceController : ControllerBase
+    {
+        public BalanceController(ILogger<BalanceController> logger, IBlockChainService blockChainService)
+        {
+            this.Logger = logger;
+            this.BlockChainService = blockChainService;
+        }
+
+        private ILogger<BalanceController> Logger { get; }
+
+        private IBlockChainService BlockChainService { get; }
+
+        [HttpGet("{currency}/{address}")]
+        public async Task<ContentResult> Get(string currency, string address)
```

Comment:
It seems not return anything when error occurred inside this function. How about use `try`...`catch` statement to handle error in this function and return some error page?

Reply:
![image](https://user-images.githubusercontent.com/26626194/93733953-555d2500-fc12-11ea-859b-05d97aa956e0.png)

I applied it in d74560a . I let it provide internal server error with *.NET Core* class, [`StatusCodeResult`][status-code-result]

[status-code-result]: https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.mvc.statuscoderesult?view=aspnetcore-3.1